types/index.d.ts

```diff
@@ -0,0 +1,28 @@
+export type AllLabels = {
+  accounts: Array<string>;
+  tokens: Array<string>;
+  blocks: ReadonlyArray<string>;
+};
+export type AccountRow = {
+  address: string;
+  nameTag: string;
+};
+export type TokenRow = {
+  address: string;
+  tokenName: string;
+  tokenSymbol: string;
+  website: string;
+};
+export type ApiResponse = {
+  d: {
+    data: Array<{
+      tokenName: string;
+      tokenSymbol: string;
+      // tokenImage: string;
```

Comment:
@dawsbot i commented the tokenImage field out as we do not have support for any chains yet. I would like to purpose once all chains have base functionality (pulling all other fields) that we add tokenImages as a new PR in one clean sweep. this is because it seems we shouldn't want just one field to be optional and do this process chain by chain involves having it optional to avoid linting errors and pre commit issues